### PR TITLE
Xode_suggestions_2021_12_19

### PR DESCRIPTION
**Pull request by Volvo-hackangel**

### CODE REVIEW

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_9.txt [hunk 1]
```diff
@@ -69,7 +69,7 @@ If you want to use a phone, use the one provided in the lobby. If you want to do
 Ending the conversation, Akiha walks out of the dining room.
 <Page20>
 I'm standing there in shock for the umpteenth time today, and the biggest one yet.
-It's not like I'm a sore loser, but I'm not the type of person who uses a mobile phone that much.//負け惜しみではないが、俺はそこまで携帯を愛用するタイプではない。
+I don't want to sound like a sore loser, but I'm not the type of person who loves my phone that much.//負け惜しみではないが、俺はそこまで携帯を愛用するタイプではない。
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_9.txt:72]: 愛用 is more like habitual user than love. I think this line was already given some TLC by Gekimon, I would revert to the original.

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_9B.txt [hunk 2]
```diff
@@ -28,7 +28,7 @@ I have many questions, but I don't know anything about Hisui and Kohaku-san at a
 "I am in charge of Shiki-sama, and my sister Kohaku is in charge of taking care of Lady Akiha. We are both in charge of the mansion while you two are away, what about it?"
 "...Taking care, you say. That's what I thought."
 <Page9>
-My shoulders are heavy with disappointment.
+My shoulders become heavy with a slump.//alt suggestions: My shoulders feel heavy.
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_9B.txt:31]: Revert.

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_9B.txt [hunk 3]
```diff
@@ -37,15 +37,15 @@ Having a girl around my own age look after me isn't to my taste right now.
 Hisui bows deeply.
 The prickling pain in my chest is brought on by my selfish guilt.
 <Page11>
-"All right. From the way Akiha puts it, she's not going to let me be alone. Besides, I don't think making her dismiss you just to make allowance for me is a good thing. So I'll accept your help but―――"//「分かった。秋葉のあの言いぶりじゃ俺を一人にはしないだろうし、そもそも俺の気持ちの問題で君を解雇させるのは良くはないと思う。ありがたくお世話になるけど―――」
+"All right. From the way Akiha puts it, she's not going to let me be alone. Besides, I don't think making her dismiss you just to make allowance for me is a good thing. So I'll accept your help but―――"//「分かった。秋葉のあの言いぶりじゃ俺を一人にはしないだろうし、そもそも俺の気持ちの問題で君を解雇させるのは良くはないと思う。ありがたくお世話になるけど―――」, what does Shiki mean by allowance here? maybe change the word allowance?
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_9B.txt:40]: Make allowances for, accommodate his preferences (not having a servant).

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_9B.txt [hunk 4]
```diff
@@ -37,15 +37,15 @@ Having a girl around my own age look after me isn't to my taste right now.
...
 <Page12>
 "Do you have any special requests?"
 "It's not that big of a deal... but um, 'Shiki-sama' is a bit too much for me.
 I'm not that kind of person, and to be honest, it's so far above me that I feel like I've been grabbed by the scruff of my neck." //I feel like there must be a better way to translate 分不相応 than "unworthy", but this isn't much better. Also, being grabbed by the neck vs by the scruff/nape of the neck has very different connotations - it's more like being carried around like a baby cat than choked.
 <Page13>
 "But, you are my master."
 "You could say that but I've been living a normal life until today.
-It's a little difficult to live a life where you've just returned home and someone your age addresses you as their master."
+It's a little difficult to adjust to a life where you've just returned home and someone your age addresses you as their master."
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_9B.txt:48]: Might rework this and above - 
> そうは言うけど、俺は今日まで普通に生きてきた身なんだ。
> 帰ってきただけなのに、同い年ぐらいの子に様づけで呼ばれる生活とか、ちょっと難しい

> You say that, but up until today I was living a normal life.
> To return here and immediately start being addressed as 'master' by a girl that's the same age as you... it's a bit much.

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_9B.txt [hunk 5]
```diff
@@ -101,7 +101,7 @@ That's how the human body works.
 <Page27>
 ...Home for the first time in seven years.
 ...Seeing my family for the first time in seven years.
-...I feel like it's someone else's house.
+...It feels like it's someone else's house.//alt suggestion: I feel like I'm in someone else's house.
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_9B.txt:104]: Dunno if we dropped the なんだか on purpose here
> ...なんだか、他人の家のように感じる自分。

**Review by rschlaikjer — CHANGES_REQUESTED** [review]

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt [hunk 6]
```diff
@@ -51,7 +51,7 @@ There is obvious contempt and provocation in Saiki's eyes, but there is no point
 <Page13>
 "Oh. Don't you have any objections?"
 "No, it's true that I abandoned my duty.
-But don't you think it's shameful for you to use words like 'deserted' and 'investment' when talking about your own family?"
+But don't you think it's shameful for you to use words like 'deserter' and 'investment' when talking about family?"//does Shiki affirm whether Saiki is a relative or not? He doesn't know currently that Saiki's are just business partners with the Tohnos for now so it may as well make sense
```
Comment (Xode672) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt:54]: Please, revert 'deserter' as per rschlaikjer's comment on line 45

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_3_2.txt [hunk 7]
```diff
@@ -32,5 +32,5 @@ Then, I was involved in an accident that left me with a physical defect, and Mak
 He said, 'Even if you're the eldest son, you can't be the successor as a weak man who might die at any time.'
 I defied his expectations and managed to recover, but by that time, my younger sister Akiha was chosen as the heir.
 <Page10>
-Up until then, Akiha had been raised as a 'Tohno daughter', but once I had dropped out, she was raised even more strictly.
-I played with Akiha in the mansion garden until I turned ten years old, and got involved in the accident. I haven't met her since then.
+Akiha was already being raised as 'A daughter of the Tohno family', but once I had dropped out, she was raised even more strictly.//それまで『遠野の娘』として育てられていた秋葉は、俺が脱落した事により、いっそう厳しく育てられる事になった。
```
Comment (yk1512) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_3_2.txt:35]: but once I dropped out, it became even more strict.

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_4_1.txt [hunk 8]
```diff
@@ -5,16 +5,16 @@ I think back to the incident with my father, Tohno Makihisa.
...
 Do not go out, do not go into the city, people are born with different statuses, protect our noble bloodline which should not be viewed as the same as the others―――
-This was the rule of Makihisa, and by extension, of the Tohno family.
+Those were the rules of Makihisa, and by extension, of the Tohno family.//Shiki just finished listing off a bunch of things that Makihisa didn't want him to do, unless "rule" is used in the sense of dominion over something. If it isn't used in the sense of dominion, then "rule" could also be changed with "principles"
 <Page4>
-I rebelled against that rule.
+I rebelled against those rules.//if my previous suggestion is accepted then please change this
 I felt that my father's teachings were wrong.
 Not because of a mistake in logic, but because of a lack of persuasiveness.
-...I'm not so sure now, but as a child, it felt like it didn't even belong in this era.//ruby dots <も|・><う|・><時|・><代|・><じ|・><ゃ|・><な|・><い|・>
+...I'm not so sure now, but as a child, it felt like it didn't even belong in this era.//ruby dots <も|・><う|・><時|・><代|・><じ|・><ゃ|・><な|・><い|・>, for italics
```
Comment (yk1512) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_4_1.txt:13]: I believe he's talking about Makihisa's teachings so the 2 its need to be changed.

I felt like they didn't even belong

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt [hunk 9]
```diff
@@ -14,15 +14,15 @@ The sudden voice made me turn around and gasp.
 In front of the door to the hallway that connects to the lobby,
 there is an all-black figure, all too strange to be real.
 <Page4>
-"Hello, Shiki-kun. Welcome to the Tohno residence...# Actually, is that wrong? In your case, it would be better to say 'welcome back'."
+"Hello, Shiki-kun. Welcome to the Tohno residence...# Actually, that would be wrong, wouldn't it? In your case, it would be better to say 'welcome back'."//alt suggestions "Actually, that's not right." or "Actually, that's not right, is it?"
```
Comment (yk1512) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt:17]: I prefer 'that's not right'

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt [hunk 10]
```diff
@@ -31,7 +31,7 @@ I wonder if there is any branch of the Tohno family with such a name.
 No, before that―――this person is, how should I say―――
 <Page8>
 "It's alright, I'm used to being looked at strangely.
-I burned my face in an accident. For a long time now, I can't go out unless I wear this. Sunlight is especially harsh.
+I burned my face in an accident. For the a long time now, I haven't been able to go outside without wearing them. Sunlight is especially harsh.
```
Comment (yk1512) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt:34]: remove 'the'

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_2.txt [hunk 11]
```diff
@@ -12,15 +12,15 @@ Just as I step into the lobby, a voice calls out to me from the stairs.
 As I look back at the large staircase leading to the second floor,
 there is an all-black figure, all too strange to be real.
 <Page4>
-"Hello, Shiki-kun. Welcome to the Tohno residence...# Actually, is that wrong? In your case, it would be better to say 'welcome back'."
+"Hello, Shiki-kun. Welcome to the Tohno residence...# Actually, that would be wrong, wouldn't it? In your case, it would be better to say 'welcome back'."//alt suggestions "Actually, that's not right." or "Actually, that's not right, is it?"
```
Comment (yk1512) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_2.txt:15]: likewise, I prefer 'that's not right'

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_7.txt [hunk 12]
```diff
@@ -23,7 +23,7 @@ The girl with long black hair says these words with a dignified look in her eyes
 My mind is completely blank.
 My thoughts have stopped entirely.
 <Page8>
-I can't even greet her properly, and I can only respond with an 'Ah' and nodding my head.
+I can't even greet her properly, and I can only respond with an 'Ah' while nodding my head.
```
Comment (yk1512) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_7.txt:26]: I can't even greet her properly, only capable of responding with an 'Ah' while nodding my head.

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_9B.txt (hunk 3 above)
Comment (yk1512) [reply script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_9B.txt:40]: > I don't think making her dismiss you just to accommodate me is a good thing

That actually reads better to me

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_9B.txt [hunk 13]
```diff
@@ -37,15 +37,15 @@ Having a girl around my own age look after me isn't to my taste right now.
...
 <Page12>
 "Do you have any special requests?"
 "It's not that big of a deal... but um, 'Shiki-sama' is a bit too much for me.
 I'm not that kind of person, and to be honest, it's so far above me that I feel like I've been grabbed by the scruff of my neck." //I feel like there must be a better way to translate 分不相応 than "unworthy", but this isn't much better. Also, being grabbed by the neck vs by the scruff/nape of the neck has very different connotations - it's more like being carried around like a baby cat than choked.
 <Page13>
 "But, you are my master."
-"You could say that but I've been living a normal life until today.
-It's a little difficult to live a life where you've just returned home and someone your age addresses you as their master."
+"You say that, but up until today I was living a normal life.//そうは言うけど、俺は今日まで普通に生きてきた身なんだ。
+To return here and immediately start being addressed as 'master' by a girl that's the same age as you... it's a bit much."//帰ってきただけなのに、同い年ぐらいの子に様づけで呼ばれる生活とか、ちょっと難しい
```
Comment (yk1512) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_9B.txt:48]: by a girl that's the same age as **me**

**Review by yk1512 — CHANGES_REQUESTED** [review]

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_3_2.txt (hunk 7 above)
Comment (rschlaikjer) [reply script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_3_2.txt:35]: Hrm, I feel like the subject for `it` is tenuous in that alternate for whatever reason. 
> Akiha was already being raised as a '...', but once I had dropped out, her upbringing became even stricter. // ?

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_9.txt [hunk 14]
```diff
@@ -34,7 +34,7 @@ Eight o'clock curfew? In this day and age? And even worse, I need someone's perm
 "Wait, wait, that's..."
 <Page10>
 "I will be setting up a study schedule for your days off.
-According to the reports, your grades are just about average, are they not?
+According to your report, your grades are just about average, are they not?
```
Comment (Gekimon) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_9.txt:37]: Does "your report" refer to "school report cards"? I don't think that's right (even though it made sense)
報告によると兄さんの成績は中の中だそうですね。
>From what I've been told, your grades are just about average, are they not?

I've looked around and I don't think 報告 can be used to mean something like school reports. There is something called 報告書 (which seems to be ~the same but less used than 調査書 and 内申書) that I think the school gives to the other schools the student want to apply to.

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_9B.txt [hunk 15]
```diff
@@ -5,7 +5,7 @@ I spent a full two hours in the dining room.
 <Page2>
 "Welcome back, Shiki-sama. Although you did not instruct me to do so, I have prepared your bed."
 <Page3>
-...she says. // reads weirdly but no fix yet
+...she says.//reads weirdly but no fix yet, Xode: should these be caps here?
```
Comment (Gekimon) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_9B.txt:8]: I think it should be caps yes?

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_9B.txt (hunk 3 above)
Comment (Gekimon) [reply script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_9B.txt]: I'm good with `accommodate`

**Review by rschlaikjer — APPROVED** [review]

**Pull request merged**